src/handlers/rfqm_handlers.ts

```diff
@@ -66,6 +84,60 @@ export class RfqmHandlers {
         res.status(HttpStatus.OK).send(indicativeQuote);
     }
 
+    public async getFirmQuoteAsync(req: express.Request, res: express.Response): Promise<void> {
+        const apiKeyLabel = req.header('0x-api-key') || 'N/A';
```

Comment:
should we just throw an error here if there's no api key?

Reply:
where do we check that the API is valid?

Reply:
regarding throwing an error: that's a good idea, I can't think of a usecase where we would allow for rfqm liquidity without a key. Will add this check

regarding the API key validity - this is some kind of hack code, but it happens in `_parseFetchIndicativeQuoteParams`, which is used by `_parseFetchFirmQuoteParams`

Reply:
cool, I would prefer both `_parseFetchFirmQuoteParams` and `_parseFetchIndicativeQuoteParams` call some third function that contains common logic, but not necessary